profile/README.md

```diff
@@ -2,19 +2,11 @@
 
 Casper strongly believes in the importance of a passionate community, and the active participation of our community members in creating innovative solutions. A robust ecosystem is essential in allowing the community to develop these solutions and projects built upon them.
 
-The Casper Network features a thriving ecosystem of projects, standards, reference implementations and tools that have evolved through active community participation.
-
-Our goal is to host an accurate, up-to-date information hub curated by the Casper core-team containing all Casper ecosystem entities in a single location to facilitate seamless community engagement.
-
 This Casper Ecosystem Organization is purpose built with a vision to serve the community. Overall, Casper provides a plethora of standards, tools and documentation for developers to easily start their projects and build applications. 
```

Comment:
This GitHub Casper Ecosystem Organization is purpose-built with a vision to serve the community. We welcome developers, entrepreneurs, and enterprises to come to build with us.

Reply:
Thanks, Piotr (@piotr-dziubecki), I have incorporated the suggested changes.